apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,29 @@ define(function(require) {
     });
   };
 });
+/**
+ * Update the sim and NFC related items
+   based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
updateSimItemsAndNfc function for telephony items is written by mozilla itself we only change its position from startup.js to panel.js because telephony items shifted from main root page to inner pages .